test/unit/SearchPagination.test.js

```diff
@@ -9,6 +9,15 @@ import SearchPagination from '../../src/app/components/SearchResults/SearchPagin
 configure({ adapter: new Adapter() });
 
 describe('Search Pagination behavior', () => {
+  describe('Get Page List', () => {
+    it('returns list of pages', () => {});
+  });
+
+  describe('calling navigateToPage changes query and calls submit', () => {
+  });
+
+  describe('Changing select calls changes searchQuery', () => {});
+
   const query = { query: 'Chicago', field: 'keyword' };
```

Comment:
I know this isn't part of the commit, but should it be wrapped in a `before` call?